src/lib/crypto/bn.cpp

```diff
@@ -39,38 +37,27 @@
 bignum_t *
 bn_bin2bn(const uint8_t *data, int len, bignum_t *ret)
 {
-    if (data == NULL) {
+    if (!data || !ret) {
```

Comment:
Just a heads up, I don't think this is right, I think the original intention here was:
* if `!data` return NULL, as that's clearly a mistake. (should be an assertion here too)
* if `!ret` allocate a `bn` and use that for the function and return it
* if `ret` then use that `bn` and return it

Reply:
@dewyatt Thanks, mislooked this. Re-pushing. Will RNP_LOG() be enough as we don't use assert's often?

Reply:
Ok, let's add an assertion as well.